server/middlewares/validations/UserVerification.js

```diff
@@ -0,0 +1,80 @@
+import jwt from 'jsonwebtoken';
```

Comment:
Why this new file. Everything here has to do with rating an article why the name `UserVerification`